[2024-07-07T13:43:29.868+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T13:43:29.923+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final.starting scheduled__2024-07-07T13:41:18.408102+00:00 [queued]>
[2024-07-07T13:43:29.943+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final.starting scheduled__2024-07-07T13:41:18.408102+00:00 [queued]>
[2024-07-07T13:43:29.948+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-07T13:43:30.028+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): starting> on 2024-07-07 13:41:18.408102+00:00
[2024-07-07T13:43:30.047+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=62) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T13:43:30.048+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final', 'starting', 'scheduled__2024-07-07T13:41:18.408102+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpiciow7au']
[2024-07-07T13:43:30.061+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask starting
[2024-07-07T13:43:30.064+0000] {standard_task_runner.py:63} INFO - Started process 64 to run task
[2024-07-07T13:43:30.170+0000] {task_command.py:426} INFO - Running <TaskInstance: Final.starting scheduled__2024-07-07T13:41:18.408102+00:00 [running]> on host 89a1261e6959
[2024-07-07T13:43:30.456+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final' AIRFLOW_CTX_TASK_ID='starting' AIRFLOW_CTX_EXECUTION_DATE='2024-07-07T13:41:18.408102+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-07T13:41:18.408102+00:00'
[2024-07-07T13:43:30.462+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T13:43:30.465+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-07T13:43:30.468+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Chuẩn bị cào dữ liệu"']
[2024-07-07T13:43:30.497+0000] {subprocess.py:86} INFO - Output:
[2024-07-07T13:43:30.502+0000] {subprocess.py:93} INFO - Chuẩn bị cào dữ liệu
[2024-07-07T13:43:30.507+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-07T13:43:30.509+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T13:43:30.595+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Final, task_id=starting, run_id=scheduled__2024-07-07T13:41:18.408102+00:00, execution_date=20240707T134118, start_date=20240707T134329, end_date=20240707T134330
[2024-07-07T13:43:30.653+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T13:43:30.702+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-07T13:43:30.705+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
